CMakeLists.txt

```diff
@@ -202,7 +202,7 @@ endif()
 
 file(GLOB_RECURSE extrafiles
     RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}
-    cmake/* fonts/* stylesheet/* xsl/*)
+    cmake/* fonts/* xsl/*)
```

Comment:
Moving the files from stylesheet causes many of your edits.
if it is not too much work, i would prefer to separate it.

The first step should only remove xsl/1.79.2/
and add the depndency docbook-xsl >= 1.79.1 to README, rpm, and the initialization of the build toolchains

Reply:
> The first step should only remove xsl/1.79.2/
> and add the dependency docbook-xsl >= 1.79.1 to README, rpm, and the initialization of the build toolchains

I'm sorry, I don't want to edit old xsl files because they are very complex.
I give up.

> The first step should only remove xsl/1.79.2/

Some cmake/* include xsl files in xsl/1.79.2/ directly such as CHM, EPUB. So it affects the build system.